RELEASE_NOTES.md

```diff
@@ -8,6 +8,17 @@
   interface allows control of how EventFlows runs tasks. Please note that
   EventFlow will only use `ITaskRunner` in very limited cases, e.g. if
   there's implantations of `ISubscribeAsynchronousTo<,,>`
+* New: `EventFlow.SQLite` now uses an `ISQLiteDatabaseMigrator` implementation 
+  to manage the creation of `EventFlow` SQL schemas when using SQLite. The 
+  behaviour and implementation is now on par with `EventFlow.MsSql` and you can 
+  use embedded resource `SQLite` scripts and the `ISQLiteDatabaseMigrator` 
+  implemenation to generate database schemas for `SQLite` read models too.
+* New: The following Nuget dependencies have been added
+  - `Helpz.SQLite` v0.2.64
```

Comment:
@JC008 Remove release notes related to updated test frameworks. The release notes are put into nuget.org descriptions and people can't download any packages related to tests